### PR TITLE
Add Stringable section

### PR DESCRIPTION
**Pull request by victor-prdh**
Hello everyone, I added Stringable and its function from English doc.

Thank you for your review !

### CODE REVIEW

File: language/predefined/stringable.xml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+<!-- {{{ Stringable intro -->
+  <section xml:id="stringable.intro">
+   &reftitle.intro;
+   <para>
+    L'interface <interfacename>Stringable</interfacename> désigne une classe possédant
+    une méthode <link linkend="object.tostring">__toString()</link>. Contrairement à la plupart des interfaces,
+    <interfacename>Stringable</interfacename> est implicitement présente sur toute classe pour
+    laquelle la méthode magique <link linkend="object.tostring">__toString()</link> est définie, et ce bien qu'elle
+    puisse et doive être déclarée explicitement.
```
Comment (devnexen) [inline language/predefined/stringable.xml:18]: `puisse et doive être déclarée explicitement` now I do not think the original is so affirmative, I would suggest
`puisse et devrait être déclarée explicitement` making it less contradictory.

Comment (victor-prdh) [reply language/predefined/stringable.xml]: yes you're right, i change it

File: language/predefined/stringable/tostring.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Retourne un la représentation d'un object sous la forme d'une &string;.
```
Comment (Girgias) [inline language/predefined/stringable/tostring.xml:27]:
```suggestion
   Retourne la représentation d'un object sous la forme d'une &string;.
```

**Comment by Girgias** [comment]
Merci!

**Pull request merged**